$NetBSD: patch-pangox.pc.in,v 1.2 2012/10/16 18:21:24 wiz Exp $

No point in a replacement library if its version is so low
it's not accepted by anything.

--- pangox.pc.in.orig	2012-08-27 23:19:32.000000000 +0000
+++ pangox.pc.in
@@ -5,7 +5,7 @@ includedir=@includedir@
 
 Name: Pango X
 Description: X Window System font support for Pango
-Version: @VERSION@
+Version: 1.30
 Requires: pango
 Libs: -L${libdir} -lpangox-1.0 @X_LIBS@
 Cflags: -I${includedir}/pango-1.0 @X_CFLAGS@
